src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/StringSyntaxAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>
+    /// <see cref="StringSyntaxAttribute"/> provides an interface to tell your IDE how a string is highlighted.
```

Comment:
```suggestion
    /// <see cref="StringSyntaxAttribute"/> tells an IDE the type of content that a string contains. It can use this to colorize or validate the contents of the string.
```
?